client/settings/payment-request-section/__tests__/index.test.js

```diff
@@ -3,9 +3,31 @@ import { screen, render } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
 import PaymentRequestSection from '..';
+import {
+	usePaymentRequestEnabledSettings
+} from '../../../data/index';
+
+jest.mock( '../../../data/index', () => ( {
```

Comment:
(minor) the `index` is not necessary

```suggestion
} from '../../../data';

jest.mock( '../../../data', () => ( {
```